src/pynorare/cli_util.py

```diff
@@ -1,17 +1,21 @@
-from pyconcepticon import Concepticon
-
-from pynorare.dataset import get_dataset_cls
+import pathlib
 
 
 def add_datasets(parser):
     parser.add_argument(
         'dataset',
-        nargs='+',
+        nargs='*',
         help='select your dataset',
         type=str)
+    parser.add_argument('--start-from', default=None)
```

Comment:
And this allows resuming `norare map` at a particular dataset. So, say, `Meier-2020` failed. You fix `Meier-2020` and run `norare map --start-from Meier`.